### PR TITLE
feat: refer to bot link syntax

### PR DESCRIPTION
**Pull request by KnorpelSenf**
https://core.telegram.org/api/links#bot-links is relevant and should be linked to

### CODE REVIEW

File: site/docs/guide/commands.md [hunk 1]
```diff
@@ -75,3 +75,6 @@ When a user clicks on the button underneath the post, their Telegram client will
 This way, your bot can identify where a user came from, and that they clicked the button underneath a specific channel post.
 
 Naturally, you can also embed such links anywhere else: on the web, in messages, in QR codes, etc.
+
+Check out this [section of the Telegram docs](https://core.telegram.org/api/links#bot-links) to see a full list of possible link formats.
```
Comment (roj1512) [inline site/docs/guide/commands.md:79]:
```suggestion
Check out [this section of the Telegram docs](https://core.telegram.org/api/links#bot-links) to see a full list of possible link formats.
```

Comment (KnorpelSenf) [reply site/docs/guide/commands.md:79]: Didn't we agree to never include __the__ in inks? I assumed this holds for __this__, __that__, __these__, and __those__, too.

**Review by roj1512 — APPROVED** [review]
Looks very perfect to me!

File: site/docs/guide/commands.md (hunk 1 above)
Comment (roj1512) [reply site/docs/guide/commands.md:79]: Right.

**Comment by KnorpelSenf** [comment]
Nice. Is anything missing before it's ready for translation?

**Comment by roj1512** [comment]
No. I just missed to add the label the moment I approved, and got busy with #578 after that.

**Review by CikiMomogi — APPROVED** [review]

**Review by habemuscode — APPROVED** [review]

**Review by roj1512 — COMMENTED** [review]
Well done bois!

**Comment by roj1512** [comment]
What feat we shippin!

**Pull request merged**